scout/server/blueprints/variant/views.py

```diff
@@ -303,19 +303,29 @@ def clinvar(institute_id, case_name, variant_id):
             conditions = request.form.getlist(key)  # can be HPO or OMIM conditions
             if conditions:
                 variant_id = key.split("@")[1]
-                cond_types = []
-                cond_values = []
+                possible_cond_types = []
+                possible_cond_values = []
 
                 for condition in conditions:
-                    cond_types.append(condition.split("_")[0])  # 'HPO' or 'OMIM'
-                    cond_values.append(condition.split("_")[1])  # HPO id or OMIM ID
+                    possible_cond_types.append(condition.split("_")[0])  # 'HPO' or 'OMIM'
```

Comment:
If you wanted to avoid some lines of code below, you could define a dictionary of possible conditions like this:

condition_types:{
"OMIM":[],
"HPO": []
}

and add terms to this dictionary here

Reply:
Agree; that is what I would have chosen from scratch. I would however have to rewrite the stuff ds of this as well to fit that data structure instead. It would be simpler in the end, but not super simple (since OMIM should still have prio). I think I will pass on the opportunity, but feel free! 😁

Reply:
Do you mind if I try this approach? We could also move a big chunk of this code to the controllers!

Reply:
Moving it to the controller would make a lot of sense! Feel very free!